sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2264,6 +2264,12 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val TRANSFORMATION_EXIT_TIMEOUT = buildConf("spark.sql.transformation.exit.timeout")
+    .internal()
+    .doc("The timeout for executor to wait for transformation script exit when EOF.")
+    .timeConf(TimeUnit.MILLISECONDS)
```

Comment:
are the users going to tune this config at the millisecond granularity? I don't think so. Maybe second is good enough here.

Reply:
ok